src/Python/tests/tests.py

```diff
@@ -17,41 +18,49 @@
 from wcon import WCONWorms, MeasurementUnit
 from wcon.measurement_unit import MeasurementUnitAtom
 
+
 class TestMeasurementUnit(unittest.TestCase):
-    def test_unit_equivalence(self):
-        self.assertTrue(MeasurementUnit.create('mm') == MeasurementUnit.create('millimetre'))
-        self.assertTrue(MeasurementUnit.create('Mm') == MeasurementUnit.create('megametre'))
-        self.assertTrue(MeasurementUnit.create('mm') != MeasurementUnit.create('Mm'))
-        self.assertTrue(MeasurementUnit.create('d') == MeasurementUnit.create('day'))
-        self.assertTrue(MeasurementUnit.create('min') == MeasurementUnit.create('minutes'))
-        self.assertTrue(MeasurementUnit.create('%') != MeasurementUnit.create('K'))
 
+    def test_unit_equivalence(self):
+        self.assertTrue(MeasurementUnit.create('mm') ==
+                        MeasurementUnit.create('millimetre'))
+        self.assertTrue(MeasurementUnit.create('Mm') ==
+                        MeasurementUnit.create('megametre'))
+        self.assertTrue(MeasurementUnit.create('mm') !=
+                        MeasurementUnit.create('Mm'))
+        self.assertTrue(MeasurementUnit.create('d') ==
+                        MeasurementUnit.create('day'))
+        self.assertTrue(MeasurementUnit.create('min') ==
+                        MeasurementUnit.create('minutes'))
+        self.assertTrue(MeasurementUnit.create(
+            '%') != MeasurementUnit.create('K'))
 
     def test_canon_conversion(self):
         self.assertTrue(MeasurementUnit.create('cm').to_canon(1) == 10)
         self.assertTrue(MeasurementUnit.create('m').to_canon(1) == 1000)
         self.assertTrue(MeasurementUnit.create('F').from_canon(0) == 32)
-        self.assertTrue(MeasurementUnit.create('microns').to_canon(1800) == 1.8)
+        self.assertTrue(MeasurementUnit.create(
+            'microns').to_canon(1800) == 1.8)
         self.assertTrue(MeasurementUnit.create('F').to_canon(32) == 0)
 
         for suf in MeasurementUnitAtom('').all_suffixes:
             # Taking the value 10 to the canonical value, then back again,
-            # should come back to 10 for all units (up to floating point 
+            # should come back to 10 for all units (up to floating point
             # epsilon)
             self.assertTrue(abs(
-                        MeasurementUnit.create(suf).from_canon(
-                        MeasurementUnit.create(suf).to_canon(10)) - 10
-                      ) < 1e-8)
-            
+                MeasurementUnit.create(suf).from_canon(
+                            MeasurementUnit.create(suf).to_canon(10)) - 10
```

Comment:
This is also considerably harder to parse than the original.  If you want to be really clear, and not bust your 80-character line length, how about

```python
self.assertTrue(
  abs(
    MeasurementUnit.create(suf).from_canon(
        MeasurementUnit.create(suf).to_canon(10)
    ) - 10
  )
)
```
though personally I'd find it a lot easier if the innards were all on one line:

```
  MeasurementUnit.create(suf).from_canon( MeasurementUnit.create(suf).to_canon(10) ) - 10
```